src/System.Drawing.Common/src/System/Drawing/BufferedGraphics.Unix.cs

```diff
@@ -112,10 +90,9 @@ public void Render(Graphics target)
         }
 
         [MonoTODO("The targetDC parameter has no equivalent in libgdiplus.")]
```

Comment:
Can we remove such MonoTODOs from the codebase in the future?
cc: @marek-safar

Reply:
@stephentoub Yes